plugins/notifications-backend-module-email/src/processor/NotificationsEmailProcessor.ts

```diff
@@ -215,20 +217,45 @@ export class NotificationsEmailProcessor implements NotificationProcessor {
     );
   }
 
-  private async sendPlainEmail(notification: Notification, emails: string[]) {
+  private getNotificationLink(notification: Notification) {
+    if (notification.payload.link) {
+      const isRelativeUrl =
+        !notification.payload.link.match(/^([a-z]*:)?\/\//i);
```

Comment:
could also try/catch around the URL constructor below, since it throws if it isn't a complete valid URL, and then always treat it as relative in the catch